libpromises/mod_custom.c

```diff
@@ -899,7 +909,7 @@ static bool PromiseModule_Validate(PromiseModule *module, const EvalContext *ctx
     PromiseModule_Send(module);
 
     // Prints errors / log messages from module:
-    JsonElement *response = PromiseModule_Receive(module, pp);
+    JsonElement *response = PromiseModule_Receive(module, pp, !dontdo /* changes_allowed */);
```

Comment:
Since this is in the validate operation, maybe we should send `changes_allowed = false`?